src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/AutoRerouteSpec.groovy

```diff
@@ -76,7 +76,6 @@ class AutoRerouteSpec extends HealthCheckSpecification {
         }
```

Comment:
Potential flaky place: 
It seems like the logic of clean-up has been changed as we switch on the port before link deletion. That can cause the auto reroute of the flow. It is a potential problem during deletion as this flow can be in an 'In progress' state. Maybe for these cases, we need to wait for the reroute completion.